lib/testing/gpuSimulateN2kCorr.cpp

```diff
@@ -82,15 +91,25 @@ void gpuSimulateN2k::main_thread() {
                                     int iy = (t * _num_local_freq + f) * _num_elements
                                              + (16 * jhi + jlo);
 
+                                    int t_rfi_in = t & 0x3FF;
+                                    int t_rfi_out = t >> 10;
+                                    int t_rfi_in_bit = t_rfi_in & 7;
+                                    int t_rfi_in_byte = t_rfi_in >> 3;
+                                    int rfi_idx =
+                                        t_rfi_in_byte + 128 * (f + _num_local_freq * t_rfi_out);
+
                                     // Decode input with the CHIME convention:
                                     // offset encoded by 8, imaginary part in
                                     // lo 4 bits, real part in high 4 bits.
                                     int xi = (input[ix] & 0x0f) - 8;
```

Comment:
There is no a type `int4x2_swapped_withoffset_t`, defined in `DataType.hpp`, which implements these operations. (There is also a type `uint1x8_t`.)